ToggleButton/ToggleButton.module.less

```diff
@@ -18,6 +18,27 @@
 			transform: @agate-togglebutton-indicator-transform;
 		}
 
+		&.huge {
+			&::after {
+				height: @agate-togglebutton-indicator-huge-height;
```

Comment:
please use the same pattern for naming variables as in other components
agate-[ComponentName]-[Size]-[RestOfName]

In this particular case, it should be @agate-togglebutton-huge-indicator-height;